ui/apps/platform/src/Containers/Policies/policies.utils.ts

```diff
@@ -447,87 +454,89 @@ function preFormatExclusionField(policy): Policy {
 /*
  * Merge client-wizard excludedDeploymentScopes and excludedImageNames properties into server exclusions property.
  */
-export function postFormatExclusionField(policy): Policy {
-    const serverPolicy: Policy = { ...policy };
+export function postFormatExclusionField(policy: ClientPolicy): Policy {
```

Comment:
Remind me, does this function need to delete client properties from the return value?
* `excludedDeploymentScopes`
* `excludedImageNames`